tensorflow/python/data/kernel_tests/checkpoint_test_base.py

```diff
@@ -0,0 +1,527 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
the file should be moved to avoid losing edit history

Reply:
So the file `tensorflow/python/data/experimental/kernel_tests/serialization/dataset_serialization_test_base.py` should be moved and renamed to `tensorflow/python/data/kernel_tests/checkpoint_test_base.py`? In addition to modifying all the tests in `experimental/kernel_tests/serialization` by importing the new `CheckpointTestBase` ?

Reply:
Correct.